server/infra/database/BaseRepository.spec.ts

```diff
@@ -69,6 +69,25 @@ describe('BaseRepository', () => {
       expect(result[0]).toHaveProperty('id', 1)
     })
 
+    it('getByFiliter with order', async () => {
+      tracker.uninstall()
+      tracker.install()
+      tracker.on('query', (query) => {
+        expect(query.sql).toMatch(/select.*testTable.*name.*order.*by.*desc.*/)
```

Comment:
Before we didn't use the complete SQL to verify the result is because that the logic is simple, now the logic become more and more complicated, so we need to check it with the complete SQL